src/signatures.js

```diff
@@ -193,4 +190,76 @@ export class Signatures {
 
     return pagelistdetails;
   }
+
+  /**
+   * Quarto (8-up) imposition that allows folding the whole printed stack at once.
+   * Because quarto pages are folded over the top, each sheet contains pages from both
+   * the inner and outer "bands". This is very different from booklet/folio page arrangement,
+   * which essentially only has a single "band" at a time.
+   */
+  quarto_booklet(pages, duplex, per_sheet, duplexrotate, sig_num) {
+    // Quarto (8-up) special case:
+    // - We need the printed stack (duplex, face-down) to fold as a whole into correct order.
+    // - We treat each "layer" j (0 = innermost, increasing outward) and build one physical sheet per layer.
+    // - For each layer we slice four 4-page "bands":
+    //     innerFront:  pages just before the fold (top half, near center)
+    //     innerBack:   pages just after the fold  (bottom half, near center)
+    //     outerFront:  pages at the front edge    (top half, near edges)
+    //     outerBack:   pages at the back edge     (bottom half, near edges)
+    // - Then we choose TL/TR/BL/BR per side via a tiny index map that alternates by layer parity.
+    // - We append layers in inner→outer order so the outermost sheet prints last and ends up on top (face-down).
+    const pagelistdetails = duplex ? [[]] : [[], []];
+    const center = pages.length / 2;
+    const pagesPerBand = 4; // 2x2 per side
+    const totalSheets = Math.floor(pages.length / per_sheet);
+
+    // Helper to push a 4-page chunk with signature flags.
+    // We put start/end marks on the back side of the outermost sheet
+    // and `isSigMiddle` on the first back position for sewing marks.
+    const pushWithFlags = (targetList, chunk, isBackSide, isOuterMost) => {
+      for (let i = 0; i < chunk.length; i++) {
+        targetList.push({
+          info: chunk[i],
+          isSigStart: isOuterMost && isBackSide && i === 0,
+          isSigEnd: isOuterMost && isBackSide && i === chunk.length - 1,
+          isSigMiddle: isBackSide && i === 0,
```

Comment:
I'll admit, I haven't scrutinized the "isSig..." values. This whole function was vibecoded with some test driven development.

If someone wants to give me pointers on how these values are used, I don't mind doing some more testing. Maybe I need to run a conversion that generates multiple signatures, then make sure they physically assemble with pages in the correct order?